### DISCUSSION TITLE
Problems using Student Developer Pack

### DISCUSSION THREAD

**Discussion by drunkguitar01** [community/community#114357] [GitHub Education]
### Select Topic Area

Question

### Body

![Q](https://github.com/community/community/assets/91728078/c866cda5-c196-41b0-95d5-e35557480210)
I am a master's student at Southeast University in China, currently navigating the wonderful resources provided through the GitHub Student Developer Pack. After successfully completing the student authentication process, I was excited to explore the capabilities of GitHub Copilot. Unfortunately, I encountered an issue where, despite the authentication being approved, I am unable to access Copilot, receiving a message that restricts my use.

I have diligently searched for a solution online but have not found any information pertinent to this specific problem. Therefore, I am reaching out to the knowledgeable GitHub community and the dedicated staff, hoping for guidance or suggestions on resolving this issue. If anyone has experienced a similar challenge or possesses insights into navigating this situation, your assistance would be immensely appreciated.

Thank you very much for taking the time to read my request. I am eager to fully engage with the tools offered by GitHub to enhance my academic and development endeavors.

**Comment by devashish2024** [comment]
It seems like it is a issue with your bank provider or billing information. Here is what you can do to try to fix it:
- Try another bank provider or account
- Try another method (UPI, etc?)
- Contact GitHub support for help

It is saying due to government restrictions, consider checking their policy too.

**Comment by klyjm** [comment]
How you to solve it? I face the same question.
